MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -64,6 +65,37 @@ def store_issue_details(issue, repo, db):
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	issue_quoted_jenkins_links = content_extracted[0]
+	issue_quoted_test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+	if issue_quoted_test_names != []:
+    		issue_details['issue_quoted_test_names'] = issue_quoted_test_names
+
+	if (jenkins_links and testnames):
+		path_list = []
+		for link in jenkins_links:
+			    for test in test_names:
+					new_string = link.split('/')
+					final_string = ''.join(new_string)
+					issue_uid2 = f'{repo_name}_{issue_number}_{final_string}_{test}.txt'
+					jenkins_output_path = f'./data/GitHubData/JenkinsOutput/{issue_uid2}'
+
+					if os.path.isfile(issue_uid2):
+						path_list.append(jenkins_output_path)
+
+					else:
+						trss_jenkins_output = query_trss_for_jenkins_output(link, test)
+						if trss_jenkins_output:
+							store_on_fs(trss_jenkins_output, jenkins_output_path)
+							path_list.append(jenkins_output_path)
+
+					if path_list:
+						issue_details['jenkins_output_path'] = path_list
```

Comment:
This can be moved out of the two loops to avoid assign value multiple times.
Also, `issue_details['jenkins_output_path']` can be renamed to `issue_details['jenkins_output_path_list']` to indicate it is a list.